<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameForm</class>
 <widget class="QWidget" name="GameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="lineButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>draw line </string>
   </property>
  </widget>
  <widget class="QPushButton" name="circleButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>draw circle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="antiButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>draw anti-aliasing line</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>421</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>listWidget</zorder>
  <zorder>lineButton</zorder>
  <zorder>circleButton</zorder>
  <zorder>antiButton</zorder>
  <zorder>label</zorder>
  <zorder>listWidget</zorder>
  <zorder></zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
